File: Programmers/Lv0/홀수vs짝수/Solution.java
package Lv0.홀수vs짝수;

class Solution {
    public int solution(int[] numList) {
        int odd = 0;
        int even = 0;

        for (int i = 0; i < numList.length; i++) {
            int index = i + 1;
            if (index % 2 == 0) {
                even += numList[i];
            } else {
                odd += numList[i];
            }
        }
        return Math.max(odd, even);
    }
}

File: Programmers/Lv0/홀수vs짝수/solution.py
def solution(num_list):
    odd = 0
    even = 0

    for i in range(len(num_list)):
        index = i + 1
        if index % 2 == 0:
            even += num_list[i]
        else:
            odd += num_list[i]

    return max(odd, even)